helm/defectdojo/templates/secret-postgresql.yaml

```diff
@@ -23,11 +23,17 @@ data:
   postgresql-postgres-password: {{ $postgresRandomPassword }}
   {{ .Values.postgresql.secretKey }}: {{ $postgresRandomPassword }}
 {{- end }}
+{{- else }}
```

Comment:
Can't we just remove the if query, code seems to be identical in both cases?

Reply:
That's what I was thinking too.